source/PaginationTrait.php

```diff
@@ -25,6 +27,9 @@
      */
     public function pagination($total, $current, $perPage = null, $neighbours = null): Pagination
     {
-        return $this['pagination']($total, $current, $perPage, $neighbours);
+        /** @var Pagination $pagination */
```

Comment:
Is it necessary to have a variable and `@var` annotation since the method has a method return type is specified?